selfdrive/boardd/tests/test_boardd_api.py

```diff
@@ -7,20 +7,21 @@
 from common.realtime import sec_since_boot
 from cereal import log
 import unittest
+from six.moves import range
```

Comment:
For tests, I don't think creating the list matters

Reply:
Agreed, removing six.